### ISSUE TITLE
dark background on error view when using dark theme

### ISSUE THREAD

**Issue by nilsburg** [yiisoft/yii2#19271]
### What steps will reproduce the problem?
On the latest Google Chrome version when using a dark theme on the OS (Windows 10) it sets the default page color to black instead of white. 

This makes the error page unreadable (look at the provided screenshots)

Setting the background color to white on the body element solves it. I could make a PR but I am not sure where the styles are located.

### What is the expected result?
To see the error page with white background (using Windows 10 light theme):
![image](https://user-images.githubusercontent.com/1023621/155500576-2d00f2ff-326a-45c1-aec5-2a2b4867475b.png)

### What do you get instead?
Using Windows 10 dark theme
![image](https://user-images.githubusercontent.com/1023621/155500455-2f7988a1-71d1-458e-8b3e-13ec7e1b6312.png)

### Additional info

| Q                | A
| ---------------- | ---
| Yii version      | 2.0.45
| PHP version      |  7.4
| Operating system | Windows 10
| Browser | Google Chrome  98.0.4758.102

**Comment by bizley** [comment]
I'm not sure what is going here but did you put word `dark` before `return` and expect it to work?

**Comment by nilsburg** [comment]
> Ohh, my bad, this is just to show the contrast, right?

Yes, I put the string ```dark``` on purpose to force the error page.

**Comment by WinterSilence** [comment]
error view use outdated [highlightJs](https://highlightjs.org/), replace code and any theme
